bridges/snowbridge/primitives/outbound-queue/src/v2/exporter.rs

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+use core::marker::PhantomData;
+use snowbridge_core::operating_mode::IsHalted;
+use sp_std::vec::Vec;
+use xcm::{
+	prelude::{Location, SendError, SendResult, SendXcm, Xcm, XcmHash},
+	VersionedLocation, VersionedXcm,
+};
+use xcm_builder::InspectMessageQueues;
+
+pub struct PausableExporter<Halted, InnerExporter>(PhantomData<(Halted, InnerExporter)>);
+
+impl<Halted: IsHalted, InnerExporter: SendXcm> SendXcm for PausableExporter<Halted, InnerExporter> {
+	type Ticket = InnerExporter::Ticket;
+
+	fn validate(
+		destination: &mut Option<Location>,
+		message: &mut Option<Xcm<()>>,
+	) -> SendResult<Self::Ticket> {
+		match Halted::is_halted() {
+			true => Err(SendError::Transport("router paused")),
```

Comment:
Seems like `SendError::NotApplicable` is the closest error type that suggests "no accepting this (message, destination) combination right now".

https://github.com/paritytech/polkadot-sdk/blob/e915cad42a5b92e76642b45cc4c5259134c8df58/polkadot/xcm/xcm-executor/src/traits/export.rs#L48

```suggestion
			true => Err(SendError::NotApplicable),
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/7/commits/06d09118efca5f4b7503bdab6ed4b911431a5d7a